crt_portal/static/sass/custom/form.scss

```diff
@@ -8,20 +8,12 @@ form#report-form,
 form#filters-form {
   .question--header {
     width: 100%; // IE11 fix
-    // margin-top: 0.5rem;
-    // margin-bottom: units($spacer-3x);
-    padding-top: 32px;
```

Comment:
:thinking: do we know what this padding was originally a fix for?

Reply:
@cantsin [The commit that introduced it](https://github.com/usdoj-crt/crt-portal/pull/599/commits/636e0685af8da1d531382de4eefe36c98949f4ac) was also in #559, just called "completed with 8"? I tracked it back to the original issue, where "FORM - Personal Characteristics: Spacing between header and question" was listed as the eighth checkbox, with this screenshot:
<img src="https://images.zenhubusercontent.com/5846f186b9dc3c6220230bf1/73989015-9947-4d2d-86ed-51960cd11781"/>

 The spacing issue between "Personal Characteristics" and the question header is something I've never been able to reproduce locally or have seen on prod.  After removing the padding added in that commit, everything also looked correct to me. 🤷 I'd be really curious how it looks to other folks now!

Regardless, I'm concerned about the approach taken. If extra padding _was_ needed here, the right approach would have been to use the USWDS `padding-top-4` classname inline, not to either add an inline style or (especially) to add it to a classname that's used widely across the app.

Reply:
Thanks! I just double checked, `Personal Characteristics` spacing seems good to me too. I agree with the `padding-top-4` approach too, I believe we missed this during review :disappointed: 

Thank you so much for your help with this! :bow: